sdks/python/apache_beam/runners/interactive/user_pipeline_tracker.py

```diff
@@ -34,31 +38,29 @@ class UserPipelineTracker:
   derived pipelines.
   """
   def __init__(self):
-    self._user_pipelines = set()
-    self._derived_pipelines = {}
-    self._pid_to_pipelines = {}
+    self._user_pipelines: set[beam.Pipeline] = set()
+    self._derived_pipelines: map[beam.Pipeline] = {}
+    self._pid_to_pipelines: map[beam.Pipeline] = {}
```

Comment:
these would be `Dict[str, beam.Pipeline]`, right?

Reply:
the `map`s 
Precommit has been fixed on master so your next commit should not have the botocore precommit issues

Reply:
done